testing/cloudbuild-templates/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3
+
+CMD while :; do nc -l -p $PORT -e sh -c 'echo -e "HTTP/1.1 200 OK\n\n Hello, World!"'; done
```

Comment:
This slipped in, it's the simplest Dockerfile I know that can be deployed to Cloud Run. Should we keep it in the repo to facilitate template testing?

Reply:
No, this was written by James Ward and proposed as a sample in its own right.
TODO: Add a note on Dockerfile usage.